.vscode/settings.json

```diff
@@ -9,6 +9,8 @@
         "_thisArgs",
         "_thisId",
         "_thisType",
+        "cba_disposable_replaceDisposableLauncher",
+        "cba_disposable_NormalLaunchers",
```

Comment:
Is it really necessary? I deleted it and nothing has changed.
```suggestion
        "_thisType"
```

Reply:
SQFLint will warn about these and say they are undefined when not added to ignore.

Reply:
Interesting, mine SQFLint seems to ignore them either way, even with VS restart. But if it works for you then fine. But one more thing. Shouldn't it end without `,` in line 13?

Reply:
VSCode has little bit more relaxed JSON format (JSON with Comments). So it works with comma at the end.